chapters/VK_EXT_wl_drm_lease_display/wl_drm_lease_display.txt

```diff
@@ -0,0 +1,51 @@
+[open,refpage='vkWaylandDrmLeaseDisplayEXT',desc='Gives access to a VkDisplayKHR using Wayland DRM lease',type='protos']
+--
+
+To acquire permission to directly lease a display in Vulkan from a Wayland
+compositor, call:
+
+include::{generated}/api/protos/vkWaylandDrmLeaseDisplayEXT.txt[]
+
+  * pname:physicalDevice is the handle to the physical device the display is on.
+  * pname:display A code:wl_display associated with the Wayland compositor that
+    the application wishes to acquire the display(s) from.
+  * pname:leaseDevice A wp_drm_lease_device_v1 connected to the Wayland compositor
+    that associated with the connectors.
+  * pname:pConnectorCount The number of valid pointers in the pname:pConnectors
+    array.
+  * pname:pConnectors Array of slink:VkWaylandDrmLeaseConnectorEXT to be acquired.
+
+All permissions necessary to control the display are granted to the Vulkan
+instance associated with pname:physicalDevice until the display is released, the
+Wayland connection specified by pname:display is terminated, or the compositor
+revokes the lease.
+If the Wayland server declines to grant the lease or the granted lease does not
+include the requested connectors, the call must: return the error code
+ename:VK_ERROR_INITIALIZATION_FAILED.
+If the Wayland server later revokes the lease, requests using the
+acquired slink:VkDisplayKHR resources must: fail with
+ename:VK_ERROR_SURFACE_LOST_KHR.
+
+include::{generated}/validity/protos/vkWaylandDrmLeaseDisplayEXT.txt[]
+--
+
+[open,refpage='VkWaylandDrmLeaseConnectorEXT',desc='Resource for relating Wayland connectors to VkDisplayKHRs',type='structs']
+--
+
+The sname:VkWaylandDrmLeaseConnectorEXT structure is defined as:
+
+include::{generated}/api/structs/VkWaylandDrmLeaseConnectorEXT.txt[]
+
+  * pname:pConnector A Wayland wp_drm_lease_connector_v1 to include in the
+    lease.
+  * pname:pDisplay The corresponding slink:VkDisplayKHR handle will be
+    returned here.
+
+The implementation will make a best-effort attempt to match each
```

Comment:
This language should probably be moved up to the vkWaylandDrmLeaseDisplayEXT section.

Reply:
Moved above